sync/src/relayer/block_transactions_process.rs

```diff
@@ -111,42 +94,32 @@ impl<'a> BlockTransactionsProcess<'a> {
                 );
 
                 // Request proposal
-                let proposals: Vec<_> = received_uncles
-                    .clone()
-                    .into_iter()
-                    .flat_map(|u| u.data().proposals().into_iter())
-                    .collect();
-                if let Err(err) = self.relayer.request_proposal_txs(
-                    self.nc.as_ref(),
-                    self.peer,
-                    block_hash.clone(),
-                    proposals,
-                ) {
-                    debug_target!(
-                        crate::LOG_TARGET_RELAY,
-                        "[BlockTransactionsProcess] request_proposal_txs: {}",
-                        err
+                {
+                    let proposals: Vec<_> = received_uncles
+                        .clone()
+                        .into_iter()
+                        .flat_map(|u| u.data().proposals().into_iter())
+                        .collect();
+                    self.relayer.request_proposal_txs(
+                        self.nc.as_ref(),
+                        self.peer,
+                        block_hash.clone(),
+                        proposals,
                     );
-                };
```

Comment:
Silently ignore`request_proposal_txs` error, maybe we should improve debug log rather than delete it.

Reply:
I had change the function from `request_proposal_txs(...) -> Result<(), FailureError>` to `request_proposal_txs (...);`, and print out the returned error inside. [diff code](https://github.com/nervosnetwork/ckb/pull/1856/files#diff-fd90e5b183cb29b0901377bac66658e0L156-L159)

So the error will not be ignored after changing.